requirements.txt

```diff
@@ -34,6 +34,6 @@ Fabric==1.14.0
 
 # Open Health Care packages on PyPi
 opal-observations==0.3.0
+-e git+https://github.com/openhealthcare/opal.git@search-seprate-with-9#egg=opal
```

Comment:
I'm not sure we want to deploy/merge this ?

Reply:
(yep, I'll push a fix)